### ISSUE TITLE
Vishnu missing 1 EXP after ultimate

### ISSUE THREAD

**Issue by kernigh** [madcowfred/padherder-issues#869]
I had Creator God, Vishnu, level 1, current EXP 0. I did the ultimate evolution to No.2081 Divine Creator of Order, Vishnu. Doing it in game gives him level 12, current EXP 19,563, but doing in PADherder gives him Curr XP 19,562. It's missing 1 EXP.

The calculation should add these numbers:
563 Keeper of Forest
3000 Divine Jade Mask
3000 Dub-emelit
3000 Dub-emelit
10000 Dub-mythlit

This adds up to 19563 (assuming no GREAT/SUPER bonus). PADherder calculated 19562 instead of 19563. PADherder is good at precise calculations. Let's fix this one.

**Comment by madcowfred** [comment]
This looks like a rounding error - does PAD always round .5 up? 375 x 1.5 for Keeper is 562.5, PADherder just floors the value.

**Comment by kernigh** [comment]
No, PAD doesn't always round .5 up. I tested with some junk in my monster box, PAD rounded 4219.5 down to 4219, but rounded 10237.5 up to 10238. Here's a list of junk -- different color value in PAD -- different x 1.5 -- same color value in PAD:

No.1183 Sprite Lily Lv.5 -- 2,813 -- 4219.5 -- 4,219
No.149 Keeper of Forest Lv.Max -- 375 -- 562.5 -- 563
No.900 Green Chaser Lv.7 -- 2,363 -- 3544.5 -- 3,544
No.1525 Kanetsugu Lv.1 -- 2,513 -- 3769.5 -- 3,769
No.1219 Himi Udon Lv.13 -- 6,825 -- 10237.5 -- 10,238
No.2310 Zlatorog Lv.11 -- 3,713 -- 5569.5 -- 5,569
No.786 Nebradisk Lv.7 -- 10,763 -- 16144.5 -- 16,144
No.663 Romantic Cathy Lv.1 -- 525 -- 787.5 -- 788
No.159 Amelit Lv.Max -- 75 -- 112.5 -- 113
No.76 Black Fighter Lv.Max -- 1,875 -- 2812.5 -- 2,813

Here are some fishy numbers:

No.832 Kozukura Lv.13 -- 3,413 -- 5119.5 -- 5119
No.832 Kozukura Lv.10 -- 2,625 -- 3837.5 -- 3938
No.832 Kozukura Lv.7 -- 1,838 -- 2757 -- 2756
No.832 Kozukura Lv.2 -- 525 -- 787.5 -- 788
No.832 Kozukura Lv.1 -- 263 -- 394.5 -- 394

The 263 might be 262.5 rounded up; the different color value of a Kozukura might be 262.5 times its level. So the same color value wouldn't be 263 x 1.5, it might be 262.5 x 1.5, which is 393.75, rounded up to 394. When I opened this issue, I didn't know the calculations were so messy!
